capreolus/benchmark/__init__.py

```diff
@@ -91,7 +92,8 @@ class ANTIQUE(Benchmark):
     topic_file = PACKAGE_PATH / "data" / "topics.antique.txt"
     fold_file = PACKAGE_PATH / "data" / "antique.json"
     query_type = "title"
-    relevance_level = 2
+    relevance_level = "map,P_1,P_5,P_10,P_20=2+ndcg_cut_5,ndcg_cut_10,ndcg_cut_20=1"  # 2
```

Comment:
I don't think we need to split this up by metric here. `pytrec_eval` only uses `relevance_level` for binary metrics; setting it doesn't affect nDCG etc.

```
>>> qrel = {'q1': {'d1': 0, 'd2': 1, 'd3': 2}, 'q2': {'d1': 0, 'd2': 1, 'd3': 2}}
>>> run = {'q1': {'d1': 1.0, 'd2': 0.0, 'd3': 1.5}, 'q2': {'d1': 0.0, 'd2': 1.0, 'd3': 0.5}}
>>> a=pytrec_eval.RelevanceEvaluator(qrel, {'map', 'ndcg'}, relevance_level=1)
>>> a.evaluate(run) 
{'q1': {'map': 0.8333333333333333, 'ndcg': 0.9502344167898356},
 'q2': {'map': 1.0, 'ndcg': 0.8597186998521972}}
>>> a=pytrec_eval.RelevanceEvaluator(qrel, {'map', 'ndcg'}, relevance_level=2)
>>> a.evaluate(run) 
{'q1': {'map': 1.0, 'ndcg': 0.9502344167898356},
 'q2': {'map': 0.5, 'ndcg': 0.8597186998521972}}

```

Reply:
oh that's even better, than we can also merge `relevance_level` and `train_ relevance_level `